src/app/app.component.html

```diff
@@ -307,13 +330,7 @@
 
 
       </div>
-      <div
-        class="uk-width-expand content-container uk-panel"
-        [class.nav-expanded]="navExpanded"
-        [class.nav-animating]="navAnimating"
-        [style.height]="( innerHeightWithoutMobileBar + 'px' )"
-        (click)="closeNav()"
-      >
+      <div class="uk-width-expand content-container uk-panel" [class.nav-expanded]="navExpanded" [class.nav-animating]="navAnimating" [style.height]="( innerHeightWithoutMobileBar + 'px' )" (click)="closeNav()">
```

Comment:
aaaaa i gave them newlines for a reason

Reply:
I think the latest push should fix these issues. Hopefully I got them all.